mongoengine/connection.py

```diff
@@ -1,17 +1,22 @@
 from pymongo import MongoClient, ReadPreference, uri_parser
-from mongoengine.python_support import (IS_PYMONGO_3, str_types)
+from mongoengine.python_support import (IS_PYMONGO_27, IS_PYMONGO_3, str_types)
 
 __all__ = ['ConnectionError', 'connect', 'register_connection',
            'DEFAULT_CONNECTION_NAME']
 
 
 DEFAULT_CONNECTION_NAME = 'default'
+AUTHENTICATION_MECHANISM = 'DEFAULT'
+
 if IS_PYMONGO_3:
     READ_PREFERENCE = ReadPreference.PRIMARY
 else:
     from pymongo import MongoReplicaSetClient
     READ_PREFERENCE = False
 
+if IS_PYMONGO_27:
```

Comment:
This could use a comment - why do we need to explicitly specify "MONGODB-CR" as the default authentication mechanism for PyMongo v2.7, while all the other PyMongo versions can use the "DEFAULT"?